setuptools/tests/test_windows_wrappers.py

```diff
@@ -108,7 +108,7 @@ def test_basic(self, tmpdir):
         ]
         proc = subprocess.Popen(
             cmd, stdout=subprocess.PIPE, stdin=subprocess.PIPE)
-        stdout, stderr = proc.communicate('hello\nworld\n'.encode('ascii'))
+        stdout, stderr = proc.communicate(b'hello\nworld\n')
```

Comment:
I'm not sure I agree with this change. Perhaps I'm being nitpicky, but I chose to use explicit encoding instead of bytes literals for clarity and symmetry with the output handling. Perhaps better, though, would be to use `text=True` or `universal_newlines=True` to enable text mode in the subprocess.

Reply:
See a56be0e5, where I'm experimenting with just using `text=True`.